tests/modules/aclcheck.c

```diff
@@ -190,6 +190,23 @@ int module_test_acl_category(RedisModuleCtx *ctx, RedisModuleString **argv, int
     return REDISMODULE_OK;
 }
 
+int commandBlockCheck(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    REDISMODULE_NOT_USED(argv);
+    REDISMODULE_NOT_USED(argc);
+    int error = 0;
+    int result = RedisModule_CreateCommand(ctx,"command.that.should.fail", module_test_acl_category, "", 0, 0, 0);
+    error |= (result == REDISMODULE_ERR);
+    RedisModuleCommand *parent = RedisModule_GetCommand(ctx,"block.commands.outside.onload");
+    result = RedisModule_CreateSubcommand(parent,"subcommand.that.should.fail",module_test_acl_category,"",0,0,0);
+    error |= (result == REDISMODULE_ERR);
```

Comment:
i suppose we want to fail if either of them succeeds.
if i read the code correctly, the "NOPERM" which we expect, will happen if at least one of them fails.

p.s. i'm not sure we should go to that level of detail in the testing, i would be willing to test just one of them, or none (not all 8 APIs)